cfme/tests/cli/test_appliance_cli.py

```diff
@@ -199,3 +202,35 @@ def is_failover_started(appliance):
     wait_for(is_failover_started, func_args=[apps[2]], timeout=450, handle_exception=True)
     apps[2].wait_for_evm_service()
     apps[2].wait_for_web_ui()
+
+
+def test_appliance_console_cli_configure_ipa(configured_appliance):
```

Comment:
This test is already covered by the following module, which includes parametrization for the available IPA servers:

https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/integration/test_cfme_auth.py

Reply:
@mshriver I cannot see that the module is configuring the use of IPA using the appliance_cli, which is the main thing my test is testing.

Reply:
It is, via this Appliance method, which is in the `ApplianceConsoleCli` class:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py#L183

The IPAAuthProvider class includes a method that supplies a fill dictionary that can be used with the above method:
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/auth/__init__.py#L257